setup.py

```diff
@@ -26,7 +26,15 @@
 def build_envd_if_not_found():
     if not os.path.exists("bin/envd"):
         logging.info("envd not found. Build from scratch")
-        errno = subprocess.call(["make", "build-release"])
+        try:
+            with open(".GIT_TAG_INFO") as f:
+                logging.info("Use build_tag from envd._version")
+                tag = f.read().strip()
+                errno = subprocess.call(
+                    ["GIT_TAG={}".format(tag), "make", "build-release"]
```

Comment:
It would be more explicit to use the `env=` parameter to pass the environment variable.
https://docs.python.org/3/library/subprocess.html#popen-constructor

Reply:
fixed